client/src/modules/reports/reports.routes.js

```diff
@@ -1,11 +1,40 @@
 angular.module('bhima.routes')
   .config(['$stateProvider', function ($stateProvider) {
+    // a list of all supported reported and their respective keys, this allows
+    // the ui-view to be populated with the correct report configuration form
+    /* @const */
+    var SUPPORTED_REPORTS = [
+      'report_accounts',
```

Comment:
In the future, we should figure out some way to make `SUPPORTED_REPORTS` data-driven.